Refactor into a single bookdown

kzollove opened the pull request:
The DBPB is currently a website that links to three bookdowns.
This refactor melts all components of the DPBP site into a single
bookdown and removes the contributing page. This format may
be easier to maintain.

See the updated GH pages [here]([kzollove.github.io/data-package-best-practices](https://kzollove.github.io/data-package-best-practices/))

I'm very open to comments and suggestions on this proposed change.

cgries commented:
I think this looks great! A few picky little things, maybe, I like the EDI logo better on top, and linked to our webpage. I assume the main headers will link to the rest of the text once it's merged, right? Go ahead and merge, as far as I am concerned.

kzollove commented:
@cgries thanks for the feedback. I have implemented the logo change (moved to upper corner, linked to EDI site).

Can you explain more about the main header linking to the rest of the text? I am guessing you are saying that when you click EML Best Practices, you get a small piece of text, but not the entire page?

As it is styled now, each "chapter" (i.e. Introduction, Conventions, General recs) is broken into its own page.

We do have control over how the html is broken down; we could opt for the entire EML Best Practices to be one long page, in which the chapters will link you to that part of the long page. The way I've implemented is consistent with the current website, and I think makes navigating easier. I think that the pages are way too long and unwieldy if the entire Best Practices "book" is one long page. But I am definitely open to opinions on the subject.

I hope I didnt totally miss the mark of your suggestion. Please clarify I did!

kzollove commented:
@cgries The issues with the links have been resolved!

I will leave this PR open to for ~ a week and if there are no other comments I will merge the refactored branch

kzollove commented:
Barring any last minute objections/ considerations I will merge this pull request by the end of today.
